README.md

```diff
@@ -5,7 +5,7 @@
 Airsonic
 ========
 [![Build Status](https://travis-ci.org/airsonic/airsonic.svg?branch=master)](https://travis-ci.org/airsonic/airsonic)
-[![Coverity scan](https://scan.coverity.com/projects/17971/badge.svg)](https://scan.coverity.com/projects/airsonic)
+[![Coverity scan](https://scan.coverity.com/projects/17971/badge.svg)](https://scan.coverity.com/projects/airsonic-airsonic)
```

Comment:
I think this particular line doesn't need changing at all.  Now the two links in this line point to different coverity projects.  I believe `17971` (and `.../projects/airsonic`) is the correct one, which is the same as what was originally used.

Reply:
Sigh, I really need some sleep.
Thanks for being picky ♥